### DISCUSSION TITLE
Request for Details on Example Dataset (T1 Ernie Template in MNI Space)

### DISCUSSION THREAD

**Discussion by frankatimm** [simnibs/simnibs#454] [2. General]
Hello everyone,

I’m currently working with data that requires multiple coregistration steps to prepare it for simulation, and I want to confirm that I’m using the correct T1 Ernie template in MNI space. I noticed the example dataset is available for download on the website, but I couldn’t find any additional information about its content or specifics.

Could anyone provide a description of this dataset, or point me toward resources with more detailed information? Any guidance would be greatly appreciated!

Thank you very much in advance for your help.

Best regards,
Franka

**Comment by TorgeW** [comment]
Unfortunately we don't have a complete description of the CHARM output file structure, however if you could describe what exactly you are looking for we probably can help you. 

Best 

Torge

**Reply by frankatimm** [reply]
Ok, thank you for your reply. I was looking for a T1 of Ernie in the MNI space. Can you help me with that?

**Reply by diedrim** [reply]
There is a function in SimNIBS, which does this transformation for you.

```from simnibs.utils.transformations import nifti_transform
from simnibs import SIMNIBSDIR
templateDir = os.path.join(SIMNIBSDIR, "segmentation/atlases/charm_atlas_mni/template.nii")
nifti_transform(
    os.path.join(m2m_dir, "T1.nii.gz"),
    os.path.join(m2m_dir, "toMNI/MNI2Conform_nonl.nii.gz"),
    templateDir,
    os.path.join(m2m_dir, "T1_MNI.nii.gz"),
    order=0,
)
```
where ```m2m_dir``` is your m2m_ernie folder where the T1 is. The fourth argument is the name and location where you want to place the transformed T1 in MNI space.

**Reply by frankatimm** [reply]
Thank you very much for your response! I have a follow-up question: I’m planning to embed a lesion registered in MNI space into the MNI Ernie template, with the goal of running simulations based on these images. Is it possible to perform both the tissue segmentation and simulation directly in MNI space or is there an extra step that I have to consider? 

Thank you again for your help! :)

**Reply by TorgeW** [reply]
The best way would be to transform the lesion into subject space and use it as a tissue mask to update the segmentation before meshing.
https://simnibs.github.io/simnibs/build/html/documentation/command_line/mni2subject.html
(use [--interpolation_order 0)

Here you can find how to update the segmentation. 
https://simnibs.github.io/simnibs/build/html/tutorial/advanced/fix_headmodel.html#fixheadmodel-tutorial
